pandas/core/computation/expr.py

```diff
@@ -40,6 +40,24 @@
 import pandas.io.formats.printing as printing
 
 
+def tokenize_backtick_quoted_string(token_generator):
```

Comment:
Can you add annotations for new functions?

Reply:
I did for the simple types, but a `token_generator` is a `Generator[Tuple[int, str, Tuple[int, int], Tuple[int, int], str], None, None]`, and I wonder how helpful this is, or that this is just a lot of noise.

Reply:
I think you should be able to match what's in type shed for the function, so `Generator[TokenInfo, None, None]`

Reply:
I think I understand.